cavern/src/main/java/org/opencadc/cavern/PosixIdentityManager.java

```diff
@@ -168,6 +155,8 @@ public Subject validate(Subject subject) throws NotAuthenticatedException {
     public Subject augment(Subject subject) {
         // delegate to configured IM
         IdentityManager im = AuthenticationUtil.getIdentityManager();
-        return im.validate(subject);
+        log.warn("augment: " + subject);
```

Comment:
Because I added an identityCache I want to keep the remote calls visible so I can easily see if redundant ones are being made... will remove once I'm closer to done. Especially for directory listing it becomes obvious that the cache is or isn't working as intended, but this is the only way to see that